main.js

```diff
@@ -0,0 +1,276 @@
+    possibleMoves = possibleMoves.filter((item) => item !== cellId);
+
+    event.target.innerText = "O";
+    console.log(event.target);
+
+    for (cell of cellButton) {
+      cell.removeEventListener("click", playerOTurn);
+    }
+    status.innerText = `${userName}'s move..`;
+    systemStatus = true;
+
+    hasWon(OMoves, winningCombinations);
+  }
+  play(systemStatus); // only needs to call 2player function because not active in comp scenario
+}
+
+// function for the computer to play. generates a random cell from the possibleMoves array, and populates it/removes it from possibleMoves.
+function compTurn() {
+  let randomCellInt = getRandomInt(possibleMoves.length); // generate random number
+  cellId = possibleMoves[randomCellInt]; // index possible moves to get cell to play
+  OMoves.push(cellId); // push move to moves array
+  possibleMoves = possibleMoves.filter((item) => item !== cellId); // remove move from possible moves array
+  let tempElement = document.getElementById(cellId); //target cell using id
+  tempElement.children[0].innerText = "O"; // change button's (first child of cell) text
+  status.innerText = `${userName}'s move..`; //update status
+  systemStatus = true;
+  hasWon(OMoves, winningCombinations); //check for winner
+  comPlay(systemStatus); //call function
+}
+
+// function to get a random integer for the computer to make a 'random' move
+function getRandomInt(max) {
+  return Math.floor(Math.random() * Math.floor(max));
+}
+
+//how to win..
+// winning combos that are possible (array of arrays)
+let winningCombinations = [
+  ["cell-0", "cell-1", "cell-2"],
+  ["cell-3", "cell-4", "cell-5"],
+  ["cell-6", "cell-7", "cell-8"],
+  ["cell-2", "cell-5", "cell-8"],
+  ["cell-0", "cell-3", "cell-6"],
+  ["cell-1", "cell-4", "cell-7"],
+  ["cell-0", "cell-4", "cell-8"],
+  ["cell-2", "cell-4", "cell-6"],
+];
+
+// function to determine if someone has won
+function hasWon(moves, winningCombinations) {
+  // create new array...
+  let foundResults = winningCombinations.filter(
+    // look at arrays inside winning combos array..
+    (array) =>
+      // for each small array look to see if each item is present.. if it is add it to the new array
+      array.filter((item) => {
+        // return what index in the players moves the winning combo item is.. if it exists in player moves, add to new array
+        return moves.indexOf(item) > -1;
+        // when length is three add this array to newly created array
+      }).length === 3
+  );
+
+  // if foundResults includes an array (1) of length three...
+  if (foundResults.length > 0) {
+    // announce someone won, so that comPlay isn't called
+    winStatus = true;
+    // change border color item.setAttribute("id", "winningCell"); so appropriate css is applied
+    foundResults[0].forEach((item) => {
+      tempElement = document.getElementById(item);
+      console.log(tempElement);
+      tempElement.id = "winningCell";
+    });
```

Comment:
Not going to lie, I originally came to grade this project since I knew about this sick border. . and was not disappointed! Really nice to see it up and running! A great feature and addition to the already solid overall project styling.